README.md

```diff
@@ -46,7 +46,9 @@ The artefact produced produced by the build process can be found under
 MMTk does not run standalone. You would need to integrate MMTk with a language implementation.
 You can either try out one of the VM bindings we have been working on, or implement your own binding in your VM for MMTk.
 You can also implement your own GC algorithm in MMTk, and run it with supported VMs.
-You can find an up-to-date API document for mmtk-core here: https://www.mmtk.io/mmtk-core/mmtk.
+You can find an up-to-date API document for mmtk-core here:
+* If you are trying to port MMTk to your language, check our public API: https://www.mmtk.io/mmtk-core/public-api
+* If you are trying to develop in MMTk (e.g. a new GC algorithm), check our full API: https://www.mmtk.io/mmtk-core/full-api
```

Comment:
I prefer not to call the full documentation an "API".  API stands for "application programming interface", and it is for those who develop applications (in our case, VMs).  I suggest "full documentation" and "full-doc".

Reply:
I have made some changes based on your suggestion. Can you check again?